htmd/qm/gaussian.py

```diff
@@ -0,0 +1,171 @@
+                f.write('\n')
+
+            if self.esp_points is not None:
+                f.write('@grid.dat /N\n')
+
+    def _readOutput(self, directory):
+
+        result = QMResult()
+        result.completed = True
+
+        with open(os.path.join(directory, 'g09.out')) as f:
+            fl = f.readlines()
+
+        try:
+            data = {}
+
+            for i in range(len(fl)):
+                if "Dipole moment (field-independent basis, Debye):" in fl[i]:
+                    s = fl[i + 1].split()
+                    data['dipole'] = [float(s[1]), float(s[3]), float(s[5]), float(s[7])]
+                if "Traceless Quadrupole moment (field-independent basis, Debye-Ang):" in fl[i]:
+                    s1 = fl[i + 1].split()
+                    s2 = fl[i + 2].split()
+                    data['quadrupole'] = [float(s1[1]), float(s1[3]), float(s1[5]), float(s2[1]), float(s2[3]),
+                                          float(s2[5])]
+                if "Mulliken atomic charges:" in fl[i] or "Mulliken charges:" in fl[i]:
+                    data['mulliken'] = []
+                    for j in range(self._natoms):
+                        data['mulliken'].append(float(fl[i + 2 + j].split()[2]))
+
+            for l in fl:
+                if "SCF Done:  E(RHF) = " in l:
+                    ff = l.split()
+                    data['energy'] = float(ff[4])
+                if "SCF Done:  E(RB3LYP) = " in l:
+                    ff = l.split()
+                    data['energy'] = float(ff[4])
+            i = 0
+            while i < len(fl):
+                if "Number     Number       Type             X           Y           Z" in fl[i]:
+                    i += 2
+                    data['coords'] = np.zeros((self._natoms, 3))
+                    for j in range(self._natoms):
+                        ff = fl[j + i].split()
+                        data['coords'][j, 0] = float(ff[3])
+                        data['coords'][j, 1] = float(ff[4])
+                        data['coords'][j, 2] = float(ff[5])
+                else:
+                    i += 1
+
+            i = 0
+            while i < len(fl):
+                if "               Potential          X             Y             Z" in fl[i]:
+                    i = i + 2 + self._natoms
+                    data['gridesp'] = []
+                    while not ("----" in fl[i]):
+                        ff = fl[i].split()
+                        data['gridesp'].append(float(ff[1]))
+                        i += 1
+                    data['gridesp'] = np.asarray(data['gridesp'], dtype=np.float)
+
+                else:
+                    i += 1
+
+        except:
+            data = None
+
+        if data is None:
+            result.errored = True
+        elif not ('energy' in data) or data['energy'] == 0.:
+            result.errored = True
+        else:
+            result.energy = data['energy'] * 627.509469  # Hartree to kcal
```

Comment:
can we get this value from a standard library like you did before for other values? if not, it's ok

Reply:
Fixed 9f231ac